acj/algorithms/pair_generator_acj/__init__.py

```diff
@@ -0,0 +1,178 @@
+import random
```

Comment:
Would you reuse pair_generator_acj in other algorithms? If not, maybe just put it with score_acj into the acj module so we have one module for each algorithm.